contact.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+<body>
+        <div class="container contact-page">
```

Comment:
You assigned a contact-page class to your `<div>` but never used it. It's a good practice to have something like this in your page, but if you are not going to use it, why placing it in first place? 

In this case it's completely fine to not use this class in your styles, but I feel your code would have been better to use the one you had in your about page (about-page class that was there before deletion), to style it.